src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiEdmTypeSchemaGenerator.cs

```diff
@@ -182,34 +182,34 @@ public static OpenApiSchema CreateSchema(this ODataContext context, IEdmPrimitiv
                     schema.Format = "decimal";
```

Comment:
I believe the format should be on the number schema

Reply:
instead of having the format on the main schema object, it should be set to the schema that sets the type to number